src/ImageSharp/Metadata/Profiles/Exif/ExifEncodedStringHelpers.cs

```diff
@@ -25,7 +25,13 @@ internal static class ExifEncodedStringHelpers
 
         private static ReadOnlySpan<byte> UndefinedCodeBytes => new byte[] { 0, 0, 0, 0, 0, 0, 0, 0 };
 
-        private static Encoding JIS0208Encoding => Encoding.GetEncoding(932);
+        private static Encoding JIS0208Encoding => CodePagesEncodingProvider.Instance.GetEncoding(932);
```

Comment:
[Docs](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding?view=netframework-4.7.2) seem to say the original should work. What was wrong?

Reply:
It doesn't work for me locally without `System.Text.Encoding.CodePages`. I don't know maybe it's my machine problem.

Reply:
Ah, I completely misread the page there. Yeah, looks like you definitely need the additional package.